src/menu/views/load_rom.c

```diff
@@ -165,12 +165,12 @@ static component_context_menu_t set_cic_type_context_menu = { .list = {
 static component_context_menu_t set_save_type_context_menu = { .list = {
     { .text = "Automatic", .action = set_save_type, .arg = (void *) (SAVE_TYPE_AUTOMATIC) },
     { .text = "None", .action = set_save_type, .arg = (void *) (SAVE_TYPE_NONE) },
-    { .text = "EEPROM 4kbit", .action = set_save_type, .arg = (void *) (SAVE_TYPE_EEPROM_4K) },
-    { .text = "EEPROM 16kbit", .action = set_save_type, .arg = (void *) (SAVE_TYPE_EEPROM_16K) },
-    { .text = "SRAM 256kbit", .action = set_save_type, .arg = (void *) (SAVE_TYPE_SRAM) },
+    { .text = "EEPROM 4kbit", .action = set_save_type, .arg = (void *) (SAVE_TYPE_EEPROM_4KBIT) },
+    { .text = "EEPROM 16kbit", .action = set_save_type, .arg = (void *) (SAVE_TYPE_EEPROM_16KBIT) },
+    { .text = "SRAM 256kbit", .action = set_save_type, .arg = (void *) (SAVE_TYPE_SRAM_256KBIT) },
     { .text = "SRAM 768kbit", .action = set_save_type, .arg = (void *) (SAVE_TYPE_SRAM_BANKED) },
```

Comment:
Shouldn't this be updated similarly to the label in the `format_rom_save_type`?

Reply:
https://github.com/Polprzewodnikowy/N64FlashcartMenu/pull/114/commits/bd6e4ceee5240ac5efb80b55a966f7b71579c9dd